src/components/CancelPausePopup/CancelPausePopup.js

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import InnerPopupWrapper from 'components/InnerPopupWrapper';
+import { withTranslation } from 'react-i18next';
+import labeling from 'containers/labeling';
+import Button from 'components/Button';
+import Loader from 'components/Loader';
+import { updateSwitch } from 'api';
+import checkmarkIconBase from 'assets/images/checkmarkBase';
+
+import {
+  ContentStyled,
+  TitleStyled,
+  TextStyled,
+  ButtonWrapperStyled
+} from 'components/InnerPopupWrapper/InnerPopupWrapperStyled';
+
+const CancelPausePopup = ({
+  popupData: { pendingSwitchId, baseOfferExpirationDate, baseOfferPrice },
+  hideInnerPopup,
+  updateList,
+  setSwitchDetails,
+  t
+}) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [step, setStep] = useState(1);
+
+  const planDetailsState = useSelector(state => state.planDetails);
+  const switchDetails = planDetailsState.switchDetails[pendingSwitchId];
+  const eventsPayload = {
+    pendingSwitchId,
+    fromOfferId: switchDetails && switchDetails.fromOfferId,
+    toOfferId: switchDetails && switchDetails.toOfferId
+  };
+
+  const cancelPause = async () => {
+    window.dispatchEvent(
+      new CustomEvent('MSSDK:cancel-switch-action-triggered', {
+        detail: eventsPayload
+      })
+    );
+    setIsLoading(true);
+    try {
+      const resp = await updateSwitch(pendingSwitchId);
+      if (!resp.errors.length) {
+        setIsLoading(false);
+        setSwitchDetails({ details: { pendingSwitchId }, type: 'delete' });
+        setStep(2);
+        window.dispatchEvent(
+          new CustomEvent('MSSDK:cancel-switch-action-successful', {
+            detail: eventsPayload
+          })
+        );
+      } else {
+        setIsError(true);
+        setIsLoading(false);
+        window.dispatchEvent(
+          new CustomEvent('MSSDK:cancel-switch-action-failed', {
+            detail: { ...eventsPayload, reason: resp.errors[0] }
+          })
+        );
+      }
+    } catch {
+      setIsError(true);
+      setIsLoading(false);
+      window.dispatchEvent(
+        new CustomEvent('MSSDK:cancel-switch-action-failed', {
+          detail: eventsPayload
+        })
+      );
+    }
+  };
+
+  return (
+    <InnerPopupWrapper
+      steps={2}
+      popupTitle={t('Cancel pause')}
+      currentStep={step}
+      isError={isError}
+    >
+      {step === 1 && (
+        <>
+          <ContentStyled>
+            <TitleStyled>{t('Cancel pause')}</TitleStyled>
```

Comment:
let's add a special keys for all texts

Reply:
Ok. I've added those but we have to remember that adding these keys will force people that are using our package to add translations file because there is no fallback string when t function is used with key
<img width="557" alt="image" src="https://user-images.githubusercontent.com/19149811/220064876-709f1cca-12aa-4e2b-9423-5a30bc76c53e.png">

Reply:
Okey. There is also a way to add default value when t() function is used with key:
https://www.i18next.com/translation-function/essentials#passing-a-default-value